ipn/ipnlocal/local.go

```diff
@@ -3202,6 +3202,13 @@ func (b *LocalBackend) EditPrefs(mp *ipn.MaskedPrefs) (ipn.PrefsView, error) {
 	if mp.SetsInternal() {
 		return ipn.PrefsView{}, errors.New("can't set Internal fields")
 	}
+
+	// Zeroing the ExitNodeId via localAPI must also zero the prior exit node.
+	if mp.ExitNodeIDSet && mp.ExitNodeID == "" {
+		mp.InternalExitNodePrior = ""
+		mp.InternalExitNodePriorSet = true
+	}
+
```

Comment:
This is way simpler than a new endpoint.  It's really just localAPI zeroing the ExitNodeID we care about. Should be all backwards and future compatible.   The backend is still free to set that InternalExitNodePrior to "future thing" when an ExitNode is enabled, and the UI will still zero it if the user decides Exit Nodes aren't their thing.

Reply:
Hmm @bradfitz just saw your comment there about the opacity of that internal value... The right answer seems to be "add a value the UI can read"...  And leave future considerations for the future.